### PR TITLE
add deps and fix certs

### PR DESCRIPTION
**Pull request by SteveRuble**
- [DP-1349] add yamllint to dependencies
- [DP-1350] improve DLP certificate environment
- [DP-1354] add wget dependency
- Bump version to 0.1.8

[DP-1349]: https://includedhealth.atlassian.net/browse/DP-1349?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DP-1350]: https://includedhealth.atlassian.net/browse/DP-1350?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DP-1354]: https://includedhealth.atlassian.net/browse/DP-1354?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lib/core/certificates/certs/readme.md [hunk 1]
```diff
@@ -3,3 +3,7 @@ These certificates are copied from https://github.com/ConsultingMD/paloaltonetwo
 They are used by the GlobalProtect VPN DLP MITM inspector.
 
 If these certs expire John Walsh is probably the person to talk to.
+
+The mozilla.pem file contains all the certs from
```
Comment (vanzin) [inline lib/core/certificates/certs/readme.md:7]: from...?

Comment (nguilford) [inline lib/core/certificates/certs/readme.md:7]: from where?

File: lib/core/certificates/step.sh [hunk 2]
```diff
@@ -53,9 +53,10 @@ function ih::setup::core.certificates::install() {
 
   ih::log::info "Acquiring cert bundle from Mozilla"
   curl https://curl.se/ca/cacert.pem >"$MOZILLA_PATH"
-
-  curl https://curl.se/ca/cacert.pem >"$HOME/.ih/certs/mozilla.pem"
+  # Append our DLP cerTs to the mozilla bundle.
   cat "$HOME/.ih/certs/grand_rounds_chained_ca.pem" >>"$HOME/.ih/certs/mozilla.pem"
```
Comment (vanzin) [inline lib/core/certificates/step.sh:57]: I'm a little confused by the different variables in this file. Is the file being modified here the same as `$MOZILLA_PATH`? Is this line the same as L62?

File: lib/core/certificates/certs/readme.md (hunk 1 above)
Comment (SteveRuble) [reply lib/core/certificates/certs/readme.md:7]: Derp, I went to go get the link to the document about where these certs were from, got distracted, and just left it blank. I'll fill that in.

Comment (SteveRuble) [reply lib/core/certificates/certs/readme.md:7]: Yah, https://github.com/ConsultingMD/homebrew-ih-public/pull/23#discussion_r979036967

File: lib/core/certificates/step.sh (hunk 2 above)
Comment (nguilford) [reply lib/core/certificates/step.sh:57]: Yeah, looks like this line should be replaced by L62, and L62 can be deleted. Also looks like L60 can be changed to reference `$MOZILLA_PATH`:
```
cat "$CA_PATH" >>"$MOZILLA_PATH"
npm config set cafile "$MOZILLA_PATH"
```

Comment (SteveRuble) [reply lib/core/certificates/step.sh]: Tidied this up.

File: lib/core/certificates/step.sh [hunk 3]
```diff
@@ -53,13 +53,12 @@ function ih::setup::core.certificates::install() {
 
   ih::log::info "Acquiring cert bundle from Mozilla"
   curl https://curl.se/ca/cacert.pem >"$MOZILLA_PATH"
+  # Append our DLP certs to the mozilla bundle.
+  cat "$CA_PATH" >>"$MOZILLA_PATH"
 
-  curl https://curl.se/ca/cacert.pem >"$HOME/.ih/certs/mozilla.pem"
```
Comment (vanzin) [inline lib/core/certificates/step.sh:57]: So as far as I can tell you're just deleting duplicate lines, not actually sorting out the "use $VARIABLE vs. use /full/path" differences?

**Review by vanzin — APPROVED** [review]

**Pull request merged**